.github/workflows/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: Check PR repository
```

Comment:
Please remove this from your PR, or add to your folder, or add to the report as code section
It should be outside your folder

Reply:
@Keldanovich  could you please share updates?

Reply:
done